test/MusicStore.Test/AnnouncementComponentTest.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Mvc.ModelBinding;
+using Microsoft.Data.Entity;
+using Microsoft.Framework.Caching.Memory;
+using Microsoft.Framework.DependencyInjection;
+using MusicStore.Models;
+using Xunit;
+
+namespace MusicStore.Components
+{
+    public class AnnouncementComponentTest
+    {
+        private readonly IServiceProvider _serviceProvider;
+
+        public AnnouncementComponentTest()
+        {
+            var services = new ServiceCollection();
+
+            services.AddEntityFramework()
+                      .AddInMemoryStore()
+                      .AddDbContext<MusicStoreContext>();
+
+            _serviceProvider = services.BuildServiceProvider();
+        }
+
+        [Fact]
+        public async Task AnnouncementComponent_Returns_LatestAlbum()
+        {
+            // Arrange
+            var viewContext = new ViewContext()
+            {
+                ViewData = new ViewDataDictionary(
+                    new EmptyModelMetadataProvider(),
+                    new ModelStateDictionary())
+            };
+
+            var announcementComponent = new AnnouncementComponent()
+            {
+                DbContext = _serviceProvider.GetRequiredService<MusicStoreContext>(),
+                Cache = _serviceProvider.GetRequiredService<IMemoryCache>(),
+                ViewContext = viewContext,
+                ViewData = new ViewDataDictionary(viewContext.ViewData),
+            };
+
+            var today = DateTime.UtcNow.Date;
```

Comment:
Agree, but I think I should use this here due to how the component is implemented
https://github.com/aspnet/MusicStore/blob/dev/src/MusicStore/Components/AnnouncementComponent.cs#L39-L46
Let me know if you want me to change this component so that it simply returns the latest one.